server/src/controllers/order.controller.ts

```diff
@@ -12,7 +15,41 @@ class OrderController {
     this.initRoutes();
   }
 
-  private initRoutes() {}
+  private initRoutes() {
+    this.router.get(`${this.prefix}/:id`, (req: Request, res: Response) =>
+      this.getOrderById(req, res)
+    );
+    this.router.post(this.prefix, (req: Request, res: Response) =>
+      this.createOrder(req, res)
+    );
+    this.router.put(`${this.prefix}/:id`, (req: Request, res: Response) =>
+      this.updateOrderById(req, res)
+    );
+  }
+  
+  private async getOrderById(req: Request, res: Response) {
+    let order = await this.orderService.getOrderById(req.params.id);
+    return new SuccessResult({
+      msg: Result.transformRequestOnMsg(req),
+      data: order,
+    }).handleSuccess(res);
+  }
+
+  private async updateOrderById(req: Request, res: Response) {
```

Comment:
If this function is just waiting for an Order update that adds a new status, I think it should change to something like:

addOrderStatusById